### ISSUE TITLE
PagSeguro não querem ganhar dinheiro?

### ISSUE THREAD

**Issue by gabrieldemery** [pagseguro/opencart#10]
Questiono isso porque vocês deixam de ganhar muito dinheiro ao não criar plugins OFICIAL PAGSEGURO para plataformas de e-commerce.
Se precisar de gente para desenvolver, me contrata! ;) #FicaADica

**Comment by dalvan03** [comment]
Verdade, meu cliente usa Mercado Pago, já que não tem modulo oficial para opencart 3.x! 

Meu cliente não quis aceitar a compra de um modulo paralelo que não fosse o oficial! Ele fatura mais R$100k por mês. Tão perdendo dinheiro.

**Comment by gabrieldemery** [comment]
É, de fato, estão perdendo dinheiro! Meus clientes sempre migram para
outros Gateways de Pagamentos, devido essa falta de suporte.
[image: Gabriel D'Emery :: Fale Comigo] <falecomigo@gabrieldemery.com>


Em qua, 24 de jul de 2019 às 19:04, dalvan03 <notifications@github.com>
escreveu:

**Comment by ivent** [comment]
Rpz um absurdo a 3 anos eu venho nesta pagina em busca de uma versão do Módulo pro Opencart 2.0 lançado a mais de 6 anos e até agora nada não acredito como uma empresa deste porte não lança um módulo oficial se precisar de um dev eu mesmo criu aqui só precisa me pagar uma micharia de de 000000000001% de seu faturamento diário até lá continuem perdendo dinheiro pra outros concorrentes blz 👍

**Comment by marcosandrade22** [comment]
Quem bom ler os comentários, não sei se este repositório é o oficial do PagSeguro, mas nem vou me arriscar.

**Comment by ivent** [comment]
É Oficial porém está desatualizada a mais de 10 anos, uma empresa deste tamanho não pode pagar 500 conto pra um dev criar um módulo oficial e atualizado pro Opencart? pena que não temos ainda um método de pagamento de Direita pois já tinha resolvido isso, mas em fim até lá temos que utilizar módulos de terceiros mesmo espero que o Pagseguro acorde pra vida e evolua tanto em qualidade quando em pensamento.
